src/index.tsx

```diff
@@ -284,19 +290,20 @@ class Cropper extends React.Component<CropperProps, State> {
   }
 
   onDragStart = ({ x, y }: Point) => {
-    this.dragStartPosition = { x, y }
+    this.transformAngle = this.props.parentRotationAngle || getRotationAngle(this.containerRef!.parentElement!) * Math.PI / 180
```

Comment:
I don't think that computing the parent element rotation should be built-in this application. The `parentRotationAngle` is enough for this advanced usage. Thus, we can remove the `getRotationAngle` function from the codebase.

Reply:
Moreover, to be aligned with the existing `rotation` prop, I think this property should be using degrees too. Thus, `transformCoordinateByAngle` second argument would be degrees.

Reply:
Ok, I can just leave ```parentRotationAngle``` and eliminate ```getRotationAngle``` from the codebase. 
I do agree we need to keep the dimensions in consistence, ```parentRotationAngle``` must be translated to radian.